src/main/java/frc/robot/subsystems/led/SetLEDState.java

```diff
@@ -0,0 +1,54 @@
+package frc.robot.subsystems.led;
+
+import edu.wpi.first.wpilibj2.command.CommandBase;
+import frc.robot.utils.DriverStationLogger;
+
+import static frc.robot.Robot.*;
+
+public class SetLEDState extends CommandBase {
+  private final static int kBlinkingAmount = 3000;
+
+  /**
+   * Changes the color of the LED's according to different states in the robot.
+   */
+  public SetLEDState() {
+    addRequirements(led);
+  }
+
+  @Override
+  public void execute() {
+    // Check the robots states and change the led colors accordingly
+    if (climb.getClimbPower() != 0 || climb.getHookPower() != 0) {
+      led.setColor(LEDColor.Blue);
+    } else if (mixer.isInStall()) {
+      led.setColor(LEDColor.Random);
+      DriverStationLogger.logToDS("Mixer is in stall!!!");
+    } else if (intake.getIsInStall()) {
+      led.setColor(LEDColor.Random);
+      DriverStationLogger.logToDS("Intake is in stall!!!");
+    } else if (loader.getIsInStall()) {
+      led.setColor(LEDColor.Random);
+      DriverStationLogger.logToDS("Loader is in stall!!!");
```

Comment:
Stall logging shouldn't be in setLedState
Also if one thing is in stall it won't tell you if another thing is in stall because it is else if